src/layouts/post/Post.jsx

```diff
@@ -10,15 +10,26 @@ import {useParams} from "react-router-dom";
 
 export function Post() {
     const params = useParams();
+    const postId = params.id;
+
+    const [post, setPost] = useState({})
 
-    const [post, setPost] = useState( {})
 
     useEffect(() => {
-        nordicAPI.getPost(params.id)
+        nordicAPI.getPost(postId)
             .then((res) => {
                 setPost(res.data);
             })
-    }, [])
+    }, [postId])
+
+    const options = {day: 'numeric', month: 'long', year: 'numeric',};
+    const setDatetime = post.created_at
+        ? new Date(Date.parse(post.created_at)).toLocaleDateString('en-US', options)
+        : 'wait';
+
+    const setArticleLength = post.content
```

Comment:
```suggestion
    const postDate = post.created_at
        ? new Date(Date.parse(post.created_at)).toLocaleDateString('en-US', options)
        : 'wait';

    const articleLength = post.content
```

`set` implies that it is a function or a method